R/metapredict_confusion.R

```diff
@@ -1,3 +1,7 @@
+mergeDataTable = function(sampleList, sampleMetadata) {
+  merge(data.table(sample = sampleList), sampleMetadata, by = 'sample', sort = FALSE)
```

Comment:
To make sure you internalize R's classes, I'm going to ask that you change the name of `sampleList`.